src/main/java/com/celements/web/plugin/api/CelementsWebPluginApi.java

```diff
@@ -839,14 +758,6 @@ public String getCaptchaId() {
     return new CaptchaCommand().getCaptchaId(context);
   }
 
-  /**
-   * @deprecated since 2.59 instead use {@link LayoutScriptService #useXWikiLoginLayout()}
-   */
-  @Deprecated
-  public boolean useXWikiLoginLayout() {
```

Comment:
still used in at least celements-web

Reply:
https://github.com/celements/celements-image/pull/82

Reply:
https://github.com/celements/celements-web/pull/458